### ISSUE TITLE
Accessibility issues for fd-select

### ISSUE THREAD

**Issue by nancy07-f** [SAP/fundamental-ngx#10118]
#### Is this a bug, enhancement, or feature request?
accessibility issues
#### Briefly describe your proposal.

1. The aria-expanded attribute is not allowed on this DIV **
2. This DIV has an aria-controls attribute value of 'fd-select-3-list-box', which includes one or more invalid or duplicate ids **
3. The aria-required attribute is not allowed on this DIV **
4. The aria-selected attribute is not allowed on this DIV with a role of list **
5. This button element does not have a mechanism that allows an accessible name value to be calculated

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
Angular version: 14.2.5
Fundamental library: 0.36.3-rc.23
#### Please provide relevant source code if applicable.
source code like:
```
        <fd-select ngDefaultControl [formControlName]="filterFieldNameEnum.AMOUNT_RANGE" 
 [controlTemplate]="budgetSelectTemplate">
          <fd-option [value]="amountRangeEnum.ALL">
            <span>All</span>
          </fd-option>
</fd-select>
```

html element:
`<div role="list" aria-live="polite" class="fd-select__control" tabindex="0" id="fd-select-3" aria-active="false" aria-expanded="false" aria-disabled="false" aria-readonly="false" aria-controls="fd-select-3-list-box" aria-haspopup="true" aria-required="false" title="All(75)" aria-selected="true" aria-labelledby="fd-form-label-5" data-la-highlighted-by="c7da6a87-f033-ab8c-1bf0-88cf3e838f6a-toggle"><div role="listitem" class="fd-select__text-content" aria-label="All(75)"><div _ngcontent-qlg-c334="" class="budget-template"><span>All</span><span>(75)</span></div><!----><!----></div><button fd-button="" tabindex="-1" class="fd-button fd-button--standard fd-button--transparent fd-select__button" type="button"><fd-icon role="presentation" class="sap-icon--slim-arrow-down"></fd-icon><!----><!----><!----><!----><!----></button><!----></div>`

**Comment by droshev** [comment]
@nancy07-f Can you check with the latest version?

**Comment by Renkoru** [comment]
One other issue related to accessibility issues with fd-select https://github.com/SAP/fundamental-ngx/issues/9963

**Comment by droshev** [comment]
@Renkoru Can you retest it on [this version](https://fundamental-ngx.netlify.app/#/core/select)

**Comment by droshev** [comment]
@Renkoru Any update?

**Comment by droshev** [comment]
no activities => closing the issue
